plugins/optimization-detective/detection.php

```diff
@@ -157,3 +157,23 @@ static function ( OD_URL_Metric_Group $group ): array {
 		array( 'type' => 'module' )
 	);
 }
+
+/**
+ * Create the rest request and register the endpoint routes for the URL Metrics Store.
+ *
+ * @since n.e.x.t
+ */
+function od_create_rest_url_metric_store_rest_request(): void {
+
+	$endpoint_controller = new OD_REST_URL_Metrics_Store_Endpoint();
+	$route_args          = $endpoint_controller->get_registration_args();
+
+	/**
+	 * Register the endpoint.
+	 */
+	register_rest_route(
+		$endpoint_controller::get_namespace(),
+		$endpoint_controller::get_route(),
```

Comment:
Maybe a personal preference, but feels a bit strange to invoke a static method on an instance. But also per my other comment, I think we can replace this with a const.

IMO, something like `OD_REST_URL_Metrics_Store_Endpoint::NAMESPACE` is easier to understand than `$endpoint_controller::NAMESPACE`.